src/panels/MissionsPanel/MissionContent.tsx

```diff
@@ -23,6 +23,20 @@ export function MissionContent({ missionOverview }: Props) {
     data: undefined
   });
 
```

Comment:
Make a comment here that describes that this resets the phases when the selected mission has changed 🙏